deb-package/build

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# Builds a Debian package for MXNET
+#
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Apache License, Version 2.0.
+# (See accompanying LICENSE file or copy at
+# https://www.apache.org/licenses/LICENSE-2.0.txt)
+set -xeu
+
+# Set up package
+pkgname=libmxnet
+pkgversion_number=$(git describe --tags --dirty |
+                    sed 's/^v\(.*\)$/\1/' |
+                    sed 's/-\([0-9]\+\)-\(g[0-9a-f]\+\)/+\1~\2/' |
+                    sed 's/\(~g[0-9a-f]\+\)-dirty$/-dirty\1/' |
+                    sed 's/-dirty/~dirty.'`date +%Y%m%d%H%M%S`'/'
+                   )
+pkgdistro=$(lsb_release -cs)
+pkgversion="${pkgversion_number}~${pkgdistro}"
```

Comment:
> I never knew upstreaming this was in the plans, but I wouldn't go that route.

Don't worry, it wasn't a strong plan.  I just noticed that the upstream repo already has quite a bit of packaging-related stuff in there (including a snap-package definition!), so it occurred to me it might be something that would be welcomed.